### ISSUE TITLE
Mixup between aria width + height and between numpy shapes

### ISSUE THREAD

**Issue by AntonioMacaronio** [facebookresearch/projectaria_tools#106]
Here is a potential bug:
https://github.com/facebookresearch/projectaria_tools/blob/6338ce8024dd0d64b233b918864f78fcde8d739a/projectaria_tools/utils/calibration_utils.py#L30

input_calib_width refers to the width of the image. For an image represented as a NumPy array, the number of numpy columns is equal to the width of the image, but the code writes `input_image.shape[0] != input_calib_width`, which is comparing the number of numpy rows. 

The same thing but vice versa happens on line 31 with `input_image.shape[1] != input_calib_height`

**Comment by chpeng-fb** [comment]
Hi @AntonioMacaronio,
did you try printing out the value of input_image.shape and input_calib_width/height?

**Comment by SeaOtocinclus** [comment]
@AntonioMacaronio can you provide any feedback?

**Comment by AntonioMacaronio** [comment]
Yes! I can add quick one line PR to fix this if need be (i will also note the changes down below) - since the main RGB camera has solution 1404x1404, no runtime errors arise with that camera. This only occurs with the side grayscale which have resolution 480x640. What happens is the `input_image` will be a numpy array of shape (480, 640), but the `input_calib_width` will be equal to 640, therefore the line checking `if input_image.shape[0] != input_calib_width or input_image.shape[1] != input_calib_height:` will be True and a ValueError will be raised.

Simply changing the conditiona to `if input_image.shape[1] != input_calib_width or input_image.shape[0] != input_calib_height:` fixes this error

**Comment by nickcharron** [comment]
@chpeng-fb, I think @AntonioMacaronio is correct. When image data is stored in an numpy array, calling shape on that array will return (height, width). So input_image.shape[0] returns the height. I've come across this issue just recently in my work. Since Aria images are always square, it doesn't matter, but if you run that same code on non-square images then it won't work

**Comment by SeaOtocinclus** [comment]
Thank you @AntonioMacaronio  for reporting the issue and @YLouWashU for fixing it in the codebase
